op-batcher/batcher/driver.go

```diff
@@ -263,16 +295,37 @@ func (l *BatchSubmitter) calculateL2BlockRangeToStore(ctx context.Context) (eth.
 		return eth.BlockID{}, eth.BlockID{}, fmt.Errorf("getting rollup client: %w", err)
 	}
 
-	cCtx, cancel := context.WithTimeout(ctx, l.Config.NetworkTimeout)
-	defer cancel()
+	var (
+		syncStatus *eth.SyncStatus
+		backoff    = time.Second
+		maxBackoff = time.Minute
```

Comment:
Nit: I think it's better to cap it off at 30 seconds, rather than a full minute. The sync-status request itself is quite harmless (it uses a very low amount of resources), so it doesn't have to back-off so much.

Reply:
fixed, please take another look.